.github/workflows/check_cirrus_cron.yml

```diff
@@ -71,7 +71,7 @@ jobs:
             - if: steps.cron.outputs.failures > 0
               name: Send failure notification e-mail
               # Ref: https://github.com/dawidd6/action-send-mail
-              uses: dawidd6/action-send-mail@v2.2.2
+              uses: dawidd6/action-send-mail@a80d851dc950256421f1d1d735a2dc1ef314ac8f # v2.2.2 https://api.github.com/repos/dawidd6/action-send-mail/git/commits/a80d851dc950256421f1d1d735a2dc1ef314ac8f
```

Comment:
I think it's valuable to keep line-length to a reasonable level.  While I appreciate the direct URL's purpose, I think the horizontal-scrolling cost is too high.  Would you mind trimming all these down to just the intended-tag comment?

It's easy and good-enough for us to `git rev-parse <tag> | grep <sha>` for any future changes.

Reply:
I have done that and removed the URL. PTAL. Thanks